src/sentry/api/endpoints/group_details.py

```diff
@@ -199,9 +210,13 @@ def get(self, request, group):
 
         action_list = self._get_actions(request, group)
 
-        if first_release:
+        if first_release is not None and last_release is not None:
+            first_release, last_release = self._get_first_last_release_info(
+                request, group, [first_release, last_release]
+            )
+        elif first_release is not None:
             first_release = self._get_release_info(request, group, first_release)
-        if last_release:
+        elif last_release is not None:
             last_release = self._get_release_info(request, group, last_release)
```

Comment:
Wondering if it actually makes most sense to make an endpoint for this, and switch to fetching the data from that endpoint? We'd have to leave this logic here for any other consumers, but we could stop requesting it on the issues page. That should make the page load much faster in general, since we won't be waiting at all.

Reply:
@wedamija 🤔 would it be OK to do that in a second PR to get this in first, and include those changes along with the more testing around retrieving releases?
Also to make sure I understand you correctly, add some param to this endpoint (to avoid breaking other consumers) so issues no longer retrieves releases here and add a second endpoint that's only releases?

Reply:
Yeah, a follow-up pr is fine. In general at Sentry we have a habit of putting a bunch of things in a single endpoint, which inevitably ends up making the endpoint really slow over time. Since this data fits in a separate subsection on the issues page, that section could just be powered by this new endpoint. I think this could be something like `groups/<id>/releases` or something like that?

In terms of the parameter, I'd call it `includeReleaseInfo` or something along those lines and default it to 1/True.

Reply:
(IMO we should do this for most related attributes in this page, like `get_group_tag_keys`, the rollup stats, etc, but that's likely a larger task)